native/NativeSampleApps/AppConfigurator/src/com/salesforce/samples/appconfigurator/AppConfiguratorState.java

```diff
@@ -45,7 +45,8 @@
 		ManagedAppCallbackURL,
 		RequireCertAuth,
 		ManagedAppCertAlias,
-        OnlyShowAuthorizedHosts
+        OnlyShowAuthorizedHosts,
```

Comment:
The other updates are to the sample app that can be used to simulate MDM and display the settings.